lib/coach/request_benchmark.rb

```diff
@@ -22,13 +22,15 @@ def notify(name, start, finish)
     end
 
     def complete(start, finish)
+      @start = start
       @duration = finish - start
     end
 
     # Serialize the results of the benchmarking
     def stats
       {
         endpoint_name: @endpoint_name,
+        started_at: @start,
```

Comment:
I went with `started_at` rather than `start` because it isn't clear that it's an absolute, rather than relative time.